models/mealtype.js

```diff
@@ -9,6 +9,7 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      MealType.belongsToMany(models.Site, { through: models.SiteMealType });
```

Comment:
This is how we tell the model in Sequelize about the relationship. It then will expect the columns in the various tables to match the expected ModelNameId format...